CommcareTranslationChecker/CommcareTranslationChecker.py

```diff
@@ -140,6 +148,54 @@ def convertCellToOutputValueList(cell):
     return outputList, messages
 
 
+def convertCellToDict(cell):
+    """
+    Convert an Excel cell to a dict of strings with occurence. <output value...> tags are ignored. 
+    If the Excel cell contains 'jr://file/' empty dict is returned.
+    Input:
+    cell (xl.cell.cell.Cell): Cell whose contents are to be parsed
+
+    Output:
+    Dict with strings as key and occurence as value
+
+   """
+    outputDict = {}
+    x = cell.value
+    if'jr://file/' in x:
+        return outputDict
+
+    try:
+        x = re.sub(r'(?:\s)<output[^, ]*', '', x)
+        x = re.sub(r'(?:\s)value=[^, ]*', '', x)
+    except Exception as e:
+        raise FatalError("FATAL ERROR determining string values for worksheet %s cell %s : %s" %
+                         (cell.parent.title, cell.coordinate, str(e)))
+    
+    for i in x.split():
+        i = re.sub(r'[\W\_]','',i)
+        if not outputDict.get(i) and (i and i.strip()) and not (re.match(r'^[0-9]*$', i)):
```

Comment:
If a word occurs more than once, do we increment occurrence count or we ignore it.

Reply:
We take only unique elements in the dictionary. The keys are unique and value will always be 1. We do not increment count if the word occurs again.